check-return.txt

```diff
@@ -0,0 +1,12 @@
+Bugfix
+   * Failures of alternative implementations of AES or DES single-block
+     functions enabled with MBEDTLS_AES_ENCRYPT_ALT, MBEDTLS_AES_DECRYPT_ALT,
+     MBEDTLS_DES_CRYPT_ECB_ALT or MBEDTLS_DES3_CRYPT_ECB_ALT were ignored.
+     This does not concern the implementation provided with Mbed TLS,
+     where this function cannot fail, or full-module replacements with
+     MBEDTLS_AES_ALT or MBEDTLS_DES_ALT. Reported by Armelle Duboc in #1092.
+
+Changes
+   * Warn if errors from AES or DES functions are ignored. This is currently
+     supported on GCC-like compilers and on MSVC and can be configured by
+     setting MBEDTLS_CHECK_RETURN in mbedtls_config.h.
```

Comment:
I'm not seeing any documentation about it in `mbedlts_config.h`. We should either document it, or remove the end of this sentence.

Reply:
Updated mbedtls_config.h